tasks/HelpOut.tex

```diff
@@ -0,0 +1,86 @@
+\section{Help Out [Housekeeper]}
+Smart speakers have become a staple device in modern households only a few years after their introduction because they fill a niche for a general assistive device in the home. A clear goal of upcoming domestic service robot technologies is to have a device that aids users in day-to-day tasks.
+
+% \subsection{Focus}
+% This test focuses on the detection and recognition of objects and their features, as well as object manipulation.
+
+\subsection{Main Goal}
+Accept verbal instructions from home occupants and perform those tasks.
+
+\noindent\textbf{Reward:} 600pts (200 points per task)\\
+
+\subsection{Bonus rewards}
+\begin{enumerate}[nosep]
+	\item Free-form English commands. The team may ask the commands to be re-worded from the generated command, in order to test more advanced natural language processing. (100 pts, each)
+	\item Going home when instructed to do so. The robot must park itself in the home position and successfully come back when requested with, \textit {<Robot Name>, I need help!}. (50pts)
+	\item Coming when called. Robot should come to the same room as the operator when requested with, \textit {<Robot Name>, I need help!}. (50pts)
```

Comment:
If the robot really only finds the way by sound I think this is really cool (especially in @home since there are no real walls...) and it would deserve more points.

Reply:
I beefed up the bonus for this, and removed the weaker bonus.